wowchemy/layouts/partials/navbar.html

```diff
@@ -105,7 +105,7 @@
         {{end}}
 
         <li class="nav-item">
-          <a class="nav-link {{if and $highlight_active_link $is_link_in_current_path }} active{{end}}" href="{{.URL | relLangURL}}"{{ if and $is_widget_page $is_same_page }} data-target="{{$hash}}"{{ end }}{{ ($.Scratch.Get "target") | safeHTMLAttr }}>
+          <a class="nav-link {{if and $highlight_active_link $is_link_in_current_path }} active{{end}}" href="{{.URL | relLangURL}}"{{ if .Params.tooltip }} data-toggle="tooltip" data-placement="{{ .Params.tooltip_placement | default "bottom" }}" title="{{ .Params.tooltip }}"{{ end }}{{ if and $is_widget_page $is_same_page }} data-target="{{$hash}}"{{ end }}{{ ($.Scratch.Get "target") | safeHTMLAttr }}>
```

Comment:
Please remove the changes that are unrelated to adding social icons to the navbar.

Reply:
This is part of my original PR, which is adding support for tooltips to navbar items.

Reply:
I understand that but please remove the code relating to the original proposal from the PR so that the PR is isolated to the more recent proposal of the social icons navbar feature.

Reply:
Fixed